CONTRIBUTING.md

```diff
@@ -1,11 +1,11 @@
-# Contributing to Measurement Plugin Support for Python
+# Contributing to Measurement Plug-In Support for Python
```

Comment:
Similar to LabVIEW, we'd like to call the repo 'title' `Measurement Plug-In SDK for Python`.

Reply:
Ah, right. Done.